CMakeLists.txt

```diff
@@ -259,6 +259,11 @@ if(WIN32)
         list(APPEND QUIC_COMMON_DEFINES QUIC_DISABLE_RESUMPTION)
     endif()
 
+    if(QUIC_TLS STREQUAL "schannel")
```

Comment:
Also going to need to be disabled for miTLS.

Reply:
updated. I briefly look but I don't know how to make it work with miTLS at the moment. Is that something users use in real or is that really just a reference implementation?